spec/Cohort.spec.ts

```diff
@@ -0,0 +1,30 @@
+import Cohort from "../src/cohort/Cohort";
+import Student from "../src/student/Student";
+
+describe("Cohort", () => {
+  let cohort: Cohort;
+
+
+  beforeEach(() => {
+    cohort = new Cohort("Cohort 1", [1, 2, 3]);
+  });
+
+  it("stores all variables from constructor", () => {
+    expect(cohort.name).toBe("Cohort 1");
```

Comment:
I like it how you have used toBe - not seen that before.  Makes the test clearer.

Reply:
`toBe` and `toEqual` aren't that difference when comparing primitives (bool, num, string). `toEquals` checks the deep equality (ex: arrays)